src/migration-wizard/steps/connect/ConnectStep.tsx

```
@@ -1,29 +1,26 @@
-  Icon,
-  List,
-  ListItem,
-  OrderType,
-  Panel,
-  PanelHeader,
-  PanelMain,
   Stack,
   StackItem,
-  Text,
   TextContent,
+  Text,
```

ESLint

[error] Member 'Text' of the import declaration should be sorted alphabetically.

(sort-imports)